install/helm/agones/values.schema.json

```diff
@@ -213,7 +213,7 @@
                 "annotations": {
                   "type": "object",
                   "additionalProperties": {
-                    "type": "string"
+                    "type": "object"
```

Comment:
```suggestion
                    "type": "object"
                    "additionalProperties": {
                        "type": "string"
                     }
```

Reply:
The service account annotations for `sdk` is a map of maps. The service account annotations for `controller` and `allocator` are maps as defined in https://agones.dev/site/docs/installation/install-agones/helm/.

So a sample section of a values.yaml file might look like:

```
  serviceaccount:
    allocator:
      name: agones-allocator
      annotations:
        "foo": bar
    controller:
      name: agones-controller
      annotations:
        "baz": qux
    sdk:
      name: agones-sdk
      annotations:
        default:
          "annotation": value
```

In Kubernetes Annotations (and labels) are required to be strings https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/. So here we want to verify that the user only passes in string values.